### PR TITLE
Modified scrollbar 

### PR DESCRIPTION
**Pull request by urmi-256**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -70,39 +70,44 @@ html[data-theme="dark"] {
...
+    background: #d261ff;
+    box-shadow: 2px 3px 3px rgb(170, 170, 170);
+    border-radius: 150px;
+}
+::-webkit-scrollbar:hover{
+   width:17px;
+     
+  }
+ ::-webkit-scrollbar-thumb:hover{
+   background-color:#d28eff;
```
Comment (smaranjitghose) [inline assets/css/style.css:107]: define dedicated variables for this with an appropriate name like scroll-thumb-cl

File: assets/css/style.css [hunk 2]
```diff
@@ -70,39 +70,44 @@ html[data-theme="dark"] {
...
+::-webkit-scrollbar-track {
+    background-color: rgb(204, 204, 204);
+    height:100vh;
+    box-shadow: inset 1px 2px 4px rgb(170, 170, 170);
+}
+::-webkit-scrollbar-thumb {
+    border: 1.5px solid rgba(217, 45, 252, 0.945);
+    max-height: 14px;
+    width:15px;
+    background: #d261ff;
```
Comment (smaranjitghose) [inline assets/css/style.css:98]: define dedicated variables for this with an appropriate name like scroll-thumb-cl

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
- On hover, only the color of the scroll bar should change and not the width of the track.
- Scroll bar should not appear in pre-loader

File: assets/css/style.css [hunk 3]
```diff
@@ -70,39 +70,44 @@ html[data-theme="dark"] {
...
     background-color: var(--bg);
+    overflow-y: auto;
 }
 
+::-webkit-scrollbar{
+    width: 15px; 
+}
 
+::-webkit-scrollbar-track {
+    background-color: rgb(204, 204, 204);
```
Comment (smaranjitghose) [inline assets/css/style.css:90]: define dedicated variables for this with an appropriate name like scroll-thumb-track

**Review by smaranjitghose — APPROVED** [review]

File: assets/css/style.css (hunk 3 above)
Comment (urmi-256) [reply assets/css/style.css]: Yes now updated. Please do check.

File: assets/js/app.js [hunk 4]
```diff
@@ -382,9 +392,14 @@ window.addEventListener("storage", function () {
 
 // Function to remove scroll bar during preload
 $(window).on('load', function() {
-
-  $('.loader-container').fadeOut(2500, function() {
-    $('.no-scroll-preload').css('overflow', 'visible');
```
Comment (smaranjitghose) [inline assets/js/app.js:387]: Please pull the recent changes from the master branch and incorporate them into your PR

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by urmi-256** [comment]
Changes done.

**Comment by smaranjitghose** [comment]
- The scrollbar is absent on the Research Page

Resesartch


- The scroll bar still appears during the preloader (fixed in the current version on master branch)

**Comment by smaranjitghose** [comment]
I would suggest to make a fresh PR after pulling all the recent changes if required or else make a squash commit to this one itself

**Comment by smaranjitghose** [comment]
This issue was exclusively for the react version. Please have a look at the Issue Titles and Descriptions carefully.

Make sure you follow the PR Guidelines in the Contributing.MD

**Pull request closed without merging**